docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-deployment-process-compatible-with-paas-ootb-deployment-process.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Is deployment process compatible with PaaS OOTB deployment process?
```

Comment:
Does it make sense to remove this document per our conversation? Since we have deployment process described in [Deployment piplines](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/configure-deployment-pipelines/deployment-pipelines.html#deployment-pipeline-types).

Reply:
I don't think we should remove this document but we can modify its content. The purpose of this document is to point attention that the deployment process must be compatible with the Spryker Cloud approach. If this document contains duplicated information, we can remove those duplicates and add links to the original documents.